plugins/generic/datacite/DatacitePlugin.php

```diff
@@ -59,6 +60,11 @@ public function register($category, $path, $mainContextId = null)
     {
         $success = parent::register($category, $path, $mainContextId);
         if ($success) {
+            // Application is set to sandbox mode and will not run the features of plugin
+            if (Config::getVar('general', 'sandbox', false)) {
+                return false;
```

Comment:
I don't think this is where you want to intervene -- it's actually the `depositXML` function that sends the XML to the outside service. This will prevent the plugin from showing up at all, and it's just the external interaction you want to prevent.

Reply:
fixed